zebra/interface.h

```diff
@@ -314,6 +314,8 @@ extern int if_ipv6_address_install(struct interface *ifp, struct prefix *prefix,
 extern int if_ip_address_uinstall(struct interface *ifp, struct prefix *prefix);
 extern int if_shutdown(struct interface *ifp);
 extern int if_no_shutdown(struct interface *ifp);
+extern int if_arp_set(struct interface *ifp);
```

Comment:
looks like these two functions are almost identical - could you converge to one, that takes a boolean?

Reply:
other functions in interface.* are dispatched. Just used the same pattern.
Moreover, it is more readable

Reply:
well, "same pattern", we have lots of patterns that are old, and that we don't want to continue to use, eh?
for me, the duplication of code is enough that it's not clearer: it'd be better to have 20 lines once, and then just the two different "set/unset" lines.

Reply:
OK done